src/app/schedule-view/component.html

```diff
@@ -8,6 +8,19 @@
         <span class="schedule-switch" (click) = "nextSchedule()">&#9654;</span>
       </div>
       <button class="schedule-button" (click) = "clear()">Clear</button>
+      <div ngbDropdown class="d-inline-block">
+          <button class="schedule-button" id="dropdownBasic1" ngbDropdownToggle>
+            Export as ...
```

Comment:
Can we make this say "share", and then make the drop down item say "download image"